monorepo/eslint-config-sane/library.cjs

```diff
@@ -16,7 +16,7 @@ module.exports = {
       typescript: true,
     },
   },
-  ignorePatterns: ["node_modules/", "build/"],
+  ignorePatterns: ["node_modules/", "build/", "src/generatedNoCheck/"],
```

Comment:
Dont let eslint touch the generated files. We want them pure for reviewing output.